### DISCUSSION TITLE
如何在pake的 src/inject/event.js里面发起http请求

### DISCUSSION THREAD

**Discussion by a86356** [tw93/Pake#687] [Q&A]
我想要在 src/inject/event.js发起http请求，使用的是js的fetch函数，但是出现都是 这种违反头策略，如下
because it violates the following Content Security Policy directive。
但是一已经配置了，tauri.config.json里面的http模块
`
    "allowlist": {
      "all": true,
      "fs": {
        "all": true,
        "scope": ["$DOWNLOAD/*"]
      },
      "http":{
        "all": true,
        "request": true,
        "scope":[
          "http://**",
          "https://**"
        ]
      }
    }
`
请问下作者大大，这个怎么解决呢？

**Comment by tw93** [comment]
有实际case吗 我本地跑跑

**Reply by tw93** [reply]
有解法，修改一下 dangerousRemoteDomainIpcAccess 这个字段，在tauri.config.json里面，同时命令行方式也是支持这个配置的，safe-domain 这个字段
